sumpy/expansion/m2l.py

```diff
@@ -740,8 +743,25 @@ def postprocess_local_exprs(self, tgt_expansion, src_expansion, m2l_result,
         return super().postprocess_local_exprs(tgt_expansion,
             src_expansion, m2l_result, src_rscale, tgt_rscale, sac)
 
-# }}} VolumeTaylorM2LWithFFT
+    def optimize_loopy_kernel(self, knl, tgt_expansion, src_expansion):
+        # Transform the kernel so that icoeff_tgt and its duplicates
+        # become the outermost iname
```

Comment:
Should this be gated to be a GPU-only transform, or is this also helpful on the CPU?